doc/en/user/source/installation/win_binary.rst

```diff
@@ -0,0 +1,73 @@
+.. _installation_windows_bin:
+
+Windows binary
+==============
+
+.. note:: For the wizard-based installer on Windows, please see the section on the :ref:`installation_windows_installer`. For installing on Windows with an existing application server such as Tomcat, please see the :ref:`installation_war` section.
+
+An alternate way of installing GeoServer on Windows is to use the platform-independent binary. This version is a GeoServer web application bundled inside `Jetty <http://eclipse.org/jetty/>`_, a lightweight and portable application server. It has the advantages of working very similarly across all operating systems and is very simple to set up.
+
+Installation
+------------
+
+#. Make sure you have a Java Runtime Environment (JRE) installed on your system. GeoServer requires a **Java 7** environment. The Oracle JRE is preferred, but OpenJDK has been known to work adequately. You can `download JRE 7 from Oracle <http://www.oracle.com/technetwork/java/javase/downloads/>`_.
+
+   .. note:: Java 8 is not currently supported.
+
+   .. note:: For more information about Java and GeoServer, please see the section on :ref:`production_java`.
+
+#. Navigate to the `GeoServer Download page <http://geoserver.org/download>`_.
+
+#. Select the version of GeoServer that you wish to download.  If you're not sure, select `Stable <http://geoserver.org/release/stable>`_.  
+
+#. Select :guilabel:`Platform Independent Binary` on the download page.
+
+#. Download the archive and unpack to the directory where you would like the program to be located.
+
+   .. note:: A suggested location would be :file:`C:\\Program Files\\GeoServer`.
+
+Setting environment variables
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+You will need to set the ``JAVA_HOME`` environment variable if it is not already set. This is the path to your JRE such that :file:`%JAVA_HOME%\\bin\\java.exe` exists.
+
+.. note:: Java 7 is required, as of GeoServer 2.6.x Java 5 is no longer supported. For more information about Java and GeoServer, please see the section on :ref:`production_java`.

```

Comment:
This text is incomplete, like as of 2.7 java 6 is no longer supported. The section on production_java is not really the best spot for installation requirements like this ...